middleware/shutdown.js

```diff
@@ -25,6 +25,18 @@ var States = {
     DISCONNECTING: 2
 };
 
+function printDeprecation() {
+    if (/krakendev/i.test(process.env.NODE_ENV)) {
+        return;
+    }
+
+    console.warn(
+`DEPRECATION WARNING: Due to deprecation of the domain module in node.js, \
```

Comment:
nit on `\`

Reply:
Yeah, I debated that too. It escapes the line breaks. I'll convert to traditional string concats.